plugins/cache.rst

```diff
@@ -125,7 +141,9 @@ to include them. You can specify any uppercase request method which conforms to
 Cache Control Handling
 ----------------------
 
-This plugin does not cache responses with ``no-store`` or ``private`` instructions.
+By default this plugin does not cache responses with ``no-store`` or ``private`` instructions. If you need to cache
```

Comment:
please add no-cache to this list. then say that with the clientCache factory method, no-store and private will be ignored.